cmd/controller/run.go

```diff
@@ -34,8 +34,10 @@ import (
 )
 
 const (
-	PprofListenAddr       = "0.0.0.0:6060"
-	kappctrlAPIPORTEnvKey = "KAPPCTRL_API_PORT"
+	PprofListenAddr                  = "0.0.0.0:6060"
+	kappctrlAPIPORTEnvKey            = "KAPPCTRL_API_PORT"
```

Comment:
I think we still prefer these to be flags rather than env vars, see e.g. :
https://github.com/vmware-tanzu/carvel-kapp-controller/blob/develop/cmd/controller/main.go#L24